platform-coop-toolkit/pcc-framework
# Switch to NPM

## Description

This PR switches docs and tests to use NPM/package-lock.json instead of yarn/yarn.lock.

## Steps to test

1. Try documented asset/build scripts in updated readme.

**Expected behavior:** Everything works as expected.

## Additional information

Not applicable.

## Related issues

Not applicable.
